drivers/docker/handle.go

```diff
@@ -239,7 +253,7 @@ func (h *taskHandle) run() {
 		werr = fmt.Errorf("Docker container exited with non-zero exit code: %d", exitCode)
 	}
 
-	container, ierr := h.waitClient.InspectContainerWithOptions(docker.InspectContainerOptions{
+	container, ierr := h.dockerClient.InspectContainerWithOptions(docker.InspectContainerOptions{
```

Comment:
This seems to be the only other behavior change... but I'm not sure how user facing it is in practice? Should we add it to the changelog anyways?